content/help/ops/traffic-management/deploy-guidelines/index.md

```diff
@@ -0,0 +1,205 @@
+  - match:
+    ...
+{{< /text >}}
+
+You also have a `VirtualService` which routes traffic for the helloworld service to a particular subset:
+
+{{< text yaml >}}
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: helloworld
+spec:
+  hosts:
+  - helloworld.default.svc.cluster.local
+  http:
+  - route:
+    - destination:
+        host: helloworld.default.svc.cluster.local
+        subset: v1
+{{< /text >}}
+
+In this situation you will notice that requests to the helloworld service via the ingress gateway will
+not be directed to subset v1 but instead will continue to use default round-robin routing.
+
+The ingress requests are using the gateway host (e.g., `myapp.com`)
+which will activate the rules in the myapp `VirtualService` that routes to any endpoint in the helloworld service.
+On the other hand, internal requests with host `helloworld.default.svc.cluster.local` will use the
+helloworld `VirtualService` which directs traffic exclusively to subset v1.
+
+To control the traffic from the gateway, you need to include the subset rule in the myapp `VirtualService`:
+
+{{< text yaml >}}
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: myapp
+spec:
+  hosts:
+  - "myapp.com" # or maybe "*" if you are testing without DNS using the ingress-gateway IP (e.g., http://1.2.3.4/hello)
+  gateways:
+  - myapp-gateway
+  http:
+  - match:
+    - uri:
+        prefix: /hello
+    route:
+    - destination:
+        host: helloworld.default.svc.cluster.local
+        subset: v1
+  - match:
+    ...
+{{< /text >}}
+
+Alternatively, you can combine both `VirtualServices` into one unit if possible:
+
+{{< text yaml >}}
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: myapp
+spec:
+  hosts:
+  - myapp.com # cannot use "*" here since this is being combined with the mesh services
+  - helloworld.default.svc.cluster.local
+  gateways:
+  - mesh # applies internally as well as externally
+  - myapp-gateway
+  http:
+  - match:
+    - uri:
+        prefix: /hello
+      gateways:
+      - myapp-gateway #restricts this rule to apply only to ingress gateway
+    route:
+    - destination:
+        host: helloworld.default.svc.cluster.local
+        subset: v1
+  - match:
+    - gateways:
+      - mesh # applies to all services inside the mesh
+    route:
+    - destination:
+        host: helloworld.default.svc.cluster.local
+        subset: v1
+{{< /text >}}
+
+## Route rules have no effect on my application
+
+If route rules are working perfectly for the [Bookinfo](/docs/examples/bookinfo/) sample,
+but similar version routing rules have no effect on your own application, it may be that
+your Kubernetes services need to be changed slightly.
+
+Kubernetes services must adhere to certain restrictions in order to take advantage of
+Istio's L7 routing features.
+Refer to the [Requirements for Pods and Services](/docs/setup/kubernetes/spec-requirements)
```

Comment:
Pods and Services should be lowercase.  If spec-requirements has that as uppercase, we merged an error.  Can you address?